apps/collabora.sh

```diff
@@ -23,10 +19,78 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+# Check if collabora is already installed
+print_text_in_color "$ICyan" "Checking if Collabora is already installed..."
+if does_this_docker_exist 'collabora/code'
+then
+    choice=$(whiptail --radiolist "It seems like 'Collabora' is already installed.\nChoose what you want to do.\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
+    "Uninstall Collabora" "" ON \
+    "Reinstall Collabora" "" OFF 3>&1 1>&2 2>&3)
+
+    case "$choice" in
+        "Uninstall Collabora")
+            print_text_in_color "$ICyan" "Uninstalling Collabora..."
+            # Check if Collabora is previously installed
+            # If yes, then stop and prune the docker container
+            docker_prune_this 'collabora/code'
+
+            # Disable RichDocuments (Collabora App) if activated
+            if [ -d "$NC_APPS_PATH"/richdocuments ]
+            then
+                occ_command app:remove richdocuments
+            fi
+            
+            msg_box "Collabora was successfully uninstalled."
+            exit
+        ;;
+        "Reinstall Collabora")
+            print_text_in_color "$ICyan" "Reinstalling Collabora..."
+            
+            # Check if Collabora is previously installed
+            # If yes, then stop and prune the docker container
+            docker_prune_this 'collabora/code'
+            
+            # Disable RichDocuments (Collabora App) if activated
+            if [ -d "$NC_APPS_PATH"/richdocuments ]
+            then
+                occ_command app:remove richdocuments
+            fi
+        ;;
+        *)
+        ;;
+    esac
+else
+    print_text_in_color "$ICyan" "Installing Collabora..."
+fi
+
+# Check if OnlyOffice is previously installed
+# If yes, then stop and prune the docker container
+docker_prune_this 'onlyoffice/documentserver'
+
+# remove OnlyOffice-documentserver if activated
+install_if_not jq
+if occ_command_no_check app:list --output=json | jq -e '.enabled | .documentserver_community' > /dev/null
+then
+    msg_box "Removing OnlyOffice-documentserver: $1\nYou will be given the option to abort when you hit OK."
+    any_key "Press any key to continue. Press CTRL+C to abort"
+    occ_command app:remove documentserver_community
```

Comment:
This logic isn't needed reallt as `occ_command app:remove documentserver_community` will remove it if it's installed, or just say that there's nothing to remove as it's not installed anyway.

Please don't add stuff that might break later on. I mean, what if the text you are grepping for are changed later on? Then the check would fail anyway, and `occ app:remve` alrady have a check built in.

That's what I meant before.

Reply:
The check was for canceling the uninstallation of onlyoffice. But if you like it more this way, I will do so.

Reply:
this is done.